deployments/admission-controllers/scheduler/README.md

```diff
@@ -56,6 +59,28 @@ kubectl get pod task0 -o yaml
 
 you'll see the `schedulerName` has been injected with value `yunikorn`.
 
+#### Validations
+
+YuniKorn loads its configuration from a configmap called yunikorn-configs, this admission controller adds a web-hook to
+validate the update and create requests for this configmap.
+It is a safeguard to protect the scheduler not to load invalid configuration, by rejecting such requests.
```

Comment:
Can we rephrase this to clearly explain the limits of what we have now:
This validation rejects invalid configuration which otherwise would have been rejected inside the scheduler update process. Validations are currently limited to a syntax check only. The scheduler could still reject a configuration if it cannot replace the current configuration due to conflicts.

Reply:
Make sense, will replace the second section with that.